packages/cli/src/commands/translate/index.ts

```diff
@@ -16,6 +16,11 @@ const translate: CommandModule = {
         type: 'string',
         describe: 'The path to your Logto instance directory',
       })
+      .option('skip-core-check', {
```

Comment:
this option should be at the same level as the check

Reply:
You mean one level down? I thought defining here would benefit all these `create`, `sync`, `sync-keys` commands

Reply:
Per offline discussion this is not a problem. Move on.